code/game/machinery/suit_cycler.dm

```diff
@@ -372,15 +372,17 @@
 		if(!suit)
 			return
 		suit.forceMove(get_turf(src))
-		usr.put_in_hands(suit)
+		if(Adjacent(usr))
+			usr.put_in_hands(suit)
 		suit = null
```

Comment:
Can't you just check adjacency at the start of topic and return if not

Reply:
nah, there are cases where it can be handy to remotely tell Stationbounds to do Cycler stuff for you

Reply:
Yeah but the rest of the code doesn't check adjacency does it?

Reply:
![immagine](https://user-images.githubusercontent.com/13016572/86769010-bbdc9900-c04e-11ea-805c-fef1ee34a413.png)
Example, all of this stuff can still be done remotely as a human, you're better off adding an adjacency check at the start with a robot exception

Reply:
donezo